tests/test-grammar-llguidance.cpp

```diff
@@ -1086,6 +1086,65 @@ static void test_json_schema() {
         });
 }
 
+static void one_hot(llama_token_data_array & tok_arr, llama_token selected) {
```

Comment:
nit: is this supposed to be `one_shot`?

Reply:
I meant https://en.wikipedia.org/wiki/One-hot though I guess this is really logits that will eventually result in 1-hot probability